woocommerce/woocommerce
# Add missing Ukrainian Subdivisions to States.php

### All Submissions:






### Changes proposed in this Pull Request:



Replay of https://github.com/woocommerce/woocommerce/pull/34834.

Closes https://github.com/woocommerce/woocommerce/issues/34803 .

Editing states.php to add missing administrative subdivisions that were missing from original Pull Request https://github.com/woocommerce/woocommerce/pull/30230.






### How to test the changes in this Pull Request:



1. Add a product to cart, go to checkout and select Ukraine as country. 
2. Check the Oblasts correspond to CLDR: https://github.com/unicode-org/cldr/blob/release-42/common/subdivisions/en.xml#L5243




### Other information:




### FOR PR REVIEWER ONLY:
